apps/server/src/shared/testing/factory/common-cartridge-element-props.factory.ts

```diff
@@ -0,0 +1,96 @@
+import { faker } from '@faker-js/faker';
+import { CommonCartridgeElementType, CommonCartridgeVersion } from '@src/modules/common-cartridge/export';
```

Comment:
Can you please move it to modules/common-cartridge/testing/* and import it from this place in other modules if you need it. With barrel file test explicit define which can be use. Or maybe in modules/common-cartridge/index.ts as export { * as testing } from './testing';
or
import { a , b c } from './testing';
const testing = { a, b, c } 
export testing;


Each module that import common-cartrige is allow to use which you define explicit as can be used in barrel file. This is the same for production and for testing code. But the ownership of this test factories are the common-cartridge module. 
If someone not import it into his module they should not (have a need to) use this.

This is for all the shared/testing parts and the test assets.

Reply:
Done. I created modules/common-cartridge/testing and moved the test factories and assets to it. I didn't create a barrel file because we only need it within the common cartridge module.